Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_001.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@jan-kaspar - is it your intention to include all RP_Detector_Assembly_* files in one geometry scenario or have multiple scenarios per file? Note, the default geometry scenario will be picked up from a DB and cannot be modified on the fly.